tensorflow/lite/python/lite.py

```diff
@@ -1086,6 +1086,9 @@ def _validate_inference_input_output_types(self, quant_mode):
       if quant_mode.is_post_training_int16x8_quantization():
         all_types = default_types + [_dtypes.int16]
       else:
+        all_types = default_types + [_dtypes.int8, _dtypes.uint8, _dtypes.int16]
+      if (self.inference_input_type not in all_types or
```

Comment:
This seems like a bad merge. I think you want to delete lines 1090-1092.

Reply:
Fixed via suggested edits